app/controllers/organizations/staff/dashboard_controller.rb

```diff
@@ -1,11 +1,61 @@
 class Organizations::Staff::DashboardController < Organizations::BaseController
+  before_action :context_authorize!, only: %i[index incomplete_tasks overdue_tasks]
+  include Pagy::Backend
   layout "dashboard"
 
   def index
     @user = current_user
     @organization = Current.organization
     @hide_footer = true
+    pet_ids = @organization.pets.pluck(:id)
```

Comment:
Here we can use Pet.all.pluck(:id) also as queries should be scoped to the tenant already.